src/main/java/it/eng/idsa/businesslogic/service/impl/ProtocolValidationServiceImpl.java

```diff
@@ -66,7 +66,7 @@ private void findRequiredProtocols() {
 	 * @return the correct Forward-To address
 	 */
 	@Override
-	public String validateProtocol(String forwardTo, Message message) {
+	public String validateProtocol(String forwardTo, Message messageForRejection) {
```

Comment:
Nice one for making it more clear where and why variable is used for

Reply:
:+1: